src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <Description>Code Generation tool for ASP.NET Core. Contains the dotnet-aspnet-codegenerator command used for generating controllers and views. </Description>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>dotnet-aspnet-codegenerator</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageTags>aspnetcore;codegenerator;scaffolding;visualstudioweb</PackageTags>
+    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <PackageId>Microsoft.VisualStudio.Web.CodeGeneration.Tools</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
+    <Compile Include="..\Shared\**\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
+    <EmbeddedResource Include="**\*.resx" />
+    <EmbeddedResource Include="Compiler\Resources\**\*" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.Extensions.ProjectModel.Abstractions.Sources\**\*.cs"
+      Exclude="..\Microsoft.Extensions.ProjectModel.Abstractions.Sources\bin\**;..\Microsoft.Extensions.ProjectModel.Abstractions.Sources\obj\**" />
+    <Compile Include="..\Microsoft.Extensions.ProjectModel.MsBuild.Sources\**\*.cs;"
+      Exclude="..\Microsoft.Extensions.ProjectModel.MsBuild.Sources\bin\**;..\Microsoft.Extensions.ProjectModel.MsBuild.Sources\obj\**" />
+    <Compile Include="..\Microsoft.Extensions.ProjectModel.DotNet.Sources\**\*.cs"
+      Exclude="..\Microsoft.Extensions.ProjectModel.DotNet.Sources\bin\**;..\Microsoft.Extensions.ProjectModel.DotNet.Sources\obj\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk" Version="1.0.0-alpha-20161104-2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="1.0.0-preview3-004050" />
```

Comment:
Update to `1.0.0-preview3-004056`. They didn't actually release the "4050" build.